src/hbbft_rbc.erl

```diff
@@ -280,15 +280,15 @@ hash_key(Map) ->
               end, #{}, Map).
 
 serialize(#rbc_data{stripes=Stripes}=Data) when Stripes /= #{} ->
-    #{rbc_data => term_to_binary(Data#rbc_data{stripes=#{}}), stripes =>
+    #{rbc_data => Data#rbc_data{stripes=#{}}, stripes =>
```

Comment:
The formatting here makes this hard to follow, maybe put each key on a separate line?
```
#{
  rbc_data => Data#rbc_data{strips = #{}},
  stripes =>
    maps:map( ... )
}
```